e2e/validators.go

```diff
@@ -269,6 +269,21 @@ func containerdVersionValidator(version string) *LiveVMValidator {
 	}
 }
 
+func textValidator(path string, expected string) *LiveVMValidator {
```

Comment:
doesn't `FileHasContentsValidator` accomplish the same thing?

Reply:
I don't think I indented to touch this part. Probably needs to be rebased with master.

Reply:
Have removed this and replaced with FileHasContentsValidator